README.md

```diff
@@ -163,6 +166,12 @@ Verilog file lists are generated via VCS and included in the config directory fo
 5. Invoke `${CALIPTRA_ROOT}/tools/scripts/Makefile` with target 'program.hex' to produce SRAM initialization files from the firmware found in `src/integration/test_suites/${TESTNAME}`
     - E.g.: `make -f ${CALIPTRA_ROOT}/tools/scripts/Makefile program.hex`
     - NOTE: TESTNAME may also be overridden in the makefile command line invocation, e.g. `make -f ${CALIPTRA_ROOT}/tools/scripts/Makefile TESTNAME=iccm_lock program.hex`
+    - NOTE: The following macro values should be overridden to match the value provided during Hardware compilation. Default values in the Makefile are shown with each macro:
```

Comment:
- `should be` or `must be`? Presumably it is not optional? If that's correct, another option is: `Override the following macro values to...`
- Probably can lowercase `hardware`